crates/walrus-sdk/src/client/store_args.rs

```diff
@@ -37,7 +37,7 @@ pub struct StoreArgs {
     /// The metrics to use for the blob.
     pub metrics: Option<Arc<ClientMetrics>>,
     /// The optional upload relay client, that allows to store the blob via the relay.
-    pub upload_relay_client: Option<UploadRelayClient>,
+    pub upload_relay_client: Option<Arc<UploadRelayClient>>,
```

Comment:
I think `UploadRelayClient` is already Send + Sync, no? It should be safe to clone. Maybe I'm thinking about it wrong.

Reply:
I was more trying to avoid copying too much data, but looking at it again, this seems rather small and the `reqwest::Client` is already an `Arc` internally, so should be fine to clone. 👍